sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16137,13 +16137,13 @@
                   "value": ""
                 }
               ],
-              "content": "<Strong> Please be advised your partner will need to sign in to StudentAid BC and provide additional information.</Strong><br/> \r\nPlease check your email after submitting your application. An email will be sent to you with instructions for your partner to sign in. Your application cannot proceed until this information is received.\r\n<br />",
+              "content": "<Strong> Please be advised your partner will need to sign in to StudentAid BC and provide additional information.</Strong><br/> \nPartners must provide their information to StudentAid BC by logging in with a B.C. Services Card. If they are ineligible for a B.C. Services Card (e.g., if they do not have a valid SIN), select \"No\" to this question and provide information on their behalf.",
```

Comment:
very minor: I don't think we have a comma after "e.g."

Reply:
Minor also, based on the AC there is a "for" missing also.
Current.
```
(e.g., if they do not have a valid SIN)
```
Expected
```
(for e.g. if they do not have a valid SIN)
```

Reply:
I checked multiple places whether there is a comma after "e.g.". Most of them suggest that there should a comma, until this reference shows that "e.g." is not followed by a comma in British English: https://jakubmarian.com/comma-after-i-e-and-e-g/. I think we can agree on "e.g." without the comma and "for" 😎